### ISSUE TITLE
Crash in XML formatting

### ISSUE THREAD

**Issue by vbuberen** [ChuckerTeam/chucker#197]
## :loudspeaker: Describe the bug
Crash during XML formatting. Seems to be an issue with `FormatUtils`.
Here is a stacktrace:
```
Process: com.chuckerteam.chucker.sample, PID: 6681
    java.lang.RuntimeException: An error occurred while executing doInBackground()
        at android.os.AsyncTask$3.done(AsyncTask.java:354)
        at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:383)
        at java.util.concurrent.FutureTask.setException(FutureTask.java:252)
        at java.util.concurrent.FutureTask.run(FutureTask.java:271)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:764)
     Caused by: java.lang.IllegalArgumentException: Not supported: http://javax.xml.XMLConstants/property/accessExternalDTD
        at org.apache.xalan.processor.TransformerFactoryImpl.setAttribute(TransformerFactoryImpl.java:580)
        at com.chuckerteam.chucker.internal.support.FormatUtils.formatXml(FormatUtils.kt:65)
        at com.chuckerteam.chucker.internal.data.entity.HttpTransaction.formatBody(HttpTransaction.kt:194)
        at com.chuckerteam.chucker.internal.data.entity.HttpTransaction.getFormattedResponseBody(HttpTransaction.kt:208)
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment$PayloadLoaderTask.doInBackground(TransactionPayloadFragment.kt:226)
        at com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment$PayloadLoaderTask.doInBackground(TransactionPayloadFragment.kt:200)
        at android.os.AsyncTask$2.call(AsyncTask.java:333)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:245) 
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) 
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) 
        at java.lang.Thread.run(Thread.java:764) 
```

## :bomb: To Reproduce
1. Open Chucker sample.
2. Do some HTTP activity.
3. Launch Chucker UI
4. Find `GET /xml` request and click on it.
5. Change tab to `Request` or `Response` and see 💥

**Comment by cortinico** [comment]
Looks like was introduced with #120 by reducing the scope of this `try-catch` block:

https://github.com/ChuckerTeam/chucker/blob/904047019f1ae2bff47db82840145fc30380e85a/library/src/main/java/com/chuckerteam/chucker/internal/support/FormatUtils.kt#L62-L80

From what I can see, the XML parsing was always broken. Previously we were just catching all the exceptions and returning the plain String. Now we just crash as we're not catching the `IllegalArgumentException`.

It's definitely worth to investigate what is going on and fix it if you have time @vbuberen

**Comment by vbuberen** [comment]
Ok, thanks for details.

Just checked the same item in Chucker 3.0.1 - it shows empty screen on `Request` tab, but actually shows the content in `Response` (see attached screenshot).
<img width="357" alt="Screen Shot 2020-01-21 at 19 59 42" src="https://user-images.githubusercontent.com/13467769/72830072-9d453a00-3c88-11ea-8e83-f4f39d18d84b.png">

I would like to tackle this issue.

**Comment by vbuberen** [comment]
Closing due to fix #201
